docs/experiments-plus/monitor.md

```diff
@@ -23,6 +21,13 @@ To monitor the status of your experiment,
       - **p-value < 0.01 and group size differs from expected size by less than 0.1% absolute**: Warning (yellow) indicating that an imbalance is possible, but the impact to the experiment is expected to be small.  This scenario typically occurs in large web experiments (1M+ users) where small variations in performance across groups can cause a small fraction of exposures to be dropped for certain groups more than others. 
       - **p-value < 0.001 and more than 0.1% absolute deviation from expected group size**: Alert (red) meaning there is likely a problem with the experiment exposures and experiment results may not be trustworthy.   
     - **Default value type mismatch** detects if an experiment's fallback default value type has differed from the set parameter type.
+    - **Group assignment healthy** verifies that your SDKs are configured correctly and surfaces if there are a high percentage of checks with assignment reasons like "Uninitialized" or "InvalidBootstrap" which might indicate your experiment assignment is not configured correctly. You can view an hourly breakdown of assignment reason via the **View Assignment Reasons** CTA
```

Comment:
Could we link to any docs explaining these reasons more? Not sure if we have any, but it would be good to have a page that says "Uninitialized means youre checking a gate before calling statsig.initialize() ..."

Reply:
Good point @alex-statsig- let me dig around & see if we have any.

Reply:
Aha! https://docs.statsig.com/sdk/debugging 

Adding now. Thanks for the idea!